hbase-server/src/main/java/org/apache/hadoop/hbase/mob/MobUtils.java

```diff
@@ -309,7 +309,18 @@ public static void cleanExpiredMobFiles(FileSystem fs, Configuration conf, Table
           fileName = hfileLink.getOriginPath().getName();
         }
 
-        Date fileDate = parseDate(MobFileName.getDateFromName(fileName));
+        String s = String.format("%s=%s-",
```

Comment:
This should be a back reference file? Do we have some util method for parsing this file name format? Or at least let's put all these methods together? So in the future if there are some modifications we could know we should change them all.

Reply:
Make sense, let me check the code first. Thanks.